CONTRIBUTING.md

```diff
@@ -31,80 +28,34 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 1. Clone this repository.
 
-2. Rust related packages:
-   - Install toolchain providing `cargo` using [rustup](https://rustup.rs/).
-   - Install [mdBook](https://rust-lang.github.io/mdBook/guide/installation.html) and the required extension with `cargo install mdbook  mdbook-i18n-helpers mdbook-last-changed `.
-
-3. Install `scarb` using [asdf](https://asdf-vm.com/) with `asdf install`. Alternatively, you can install `scarb` manually by following the instructions [here](https://docs.swmansion.com/scarb/).
-
-## MdBook
+2. Install the required dependencies using pnpm:
+```
+pnpm i
+```
 
-All the Markdown files **MUST** be edited in english. To work locally in english:
+3. Start the development server:
+```
+pnpm dev
+```
 
-- Start a local server with `mdbook serve` and visit [localhost:3000](http://localhost:3000) to view the book.
-  You can use the `--open` flag to open the browser automatically: `mdbook serve --open`.
+## Working with Markdown Files
 
-- Make changes to the book and refresh the browser to see the changes.
+All Markdown files (*.md) MUST be edited in English. Follow these steps to work locally with Markdown files:
 
-- Open a PR with your changes.
+- Make changes to the desired .md files in your preferred text editor.
+- Save the changes, and your browser window should automatically refresh to reflect the updates.
+- Once you've finished making your changes, build the application to ensure everything works as expected:
+  ```
+  pnpm build
+  ```
+- If everything looks good, commit your changes and open a pull request with your modifications.
 
 ## Adding a new chapter
 
-To add a new chapter, create a new markdown file in the `src` directory. All the Markdown files **MUST** be edited in english. In order to add them to the book, you need to add a link to it in the `src/SUMMARY.md` file.
-
-Do not write directly Cairo program inside the markdown files. Instead, use code blocks that import the Cairo programs from the `listing` directory. These programs are bundled into scarb projects, which makes it easier to test and build all programs. See the next section for more details.
```

Comment:
Keep the `Do not write directly Cairo program inside the markdown files` paragraph.